core/bones/keyBone.py

```diff
@@ -8,8 +8,8 @@
 class keyBone(baseBone):
 	type = "key"
 
-	def __init__(self, descr="Key", readOnly=True, visible=False, **kwargs):
-		super(keyBone, self).__init__(descr=descr, readOnly=True, visible=visible, defaultValue=None, **kwargs)
+	def __init__(self, *, descr="Key", readOnly=True, visible=False, **kwargs):
+		super(keyBone, self).__init__(descr=descr, readOnly=readOnly, visible=visible, defaultValue=None, **kwargs)
```

Comment:
@tsteinruecken OK thanks. I think that only very few projects use args-notation for bones. Not even my older projects use this feature, so it might be a "small breaking change", not requiring a new minor release.

Reply:
Hello @tsteinruecken, this is correct, but might be a wanted behavior. According to https://github.com/viur-framework/viur-core/blob/main/core/skeleton.py#L580, a Skeleton's standard key is predefined as readOnly=True, and this is also the case with the keyBone's defaultValue.

Anyway, developers should not be limited to explicitly allow setting the readOnly behavior of a keyBone on their own risk.